src/core/components/ChatView.tsx

```diff
@@ -47,7 +47,11 @@ export const ChatView = ({
   const containerRef = useRef<HTMLDivElement>(null);
   const handleContentRendered = useCallback(() => {
     if (containerRef.current) {
-      containerRef.current.scrollTop = containerRef.current.scrollHeight;
+      requestAnimationFrame(() => {
```

Comment:
make sure the page scrolls to the bottom properly when completed

Reply:
@downing034 I think you can use `useScrollToBottom` here

Reply:
Ahh let me give that a go

Reply:
Locally I added rolled this back to `containerRef.current.scrollTop = containerRef.current.scrollHeight;` and then added `useScrollToBottom(handleContentRendered);` afterwards like our other usage and it continues to scroll back to the top of the ChatView scrollContent container